### PR TITLE
Update trade_file.md

### PR DESCRIPTION
**Pull request by jpriceclst**
Add exec_mpid to allocation trades, and some overdue edits to the trade spec in general

### CODE REVIEW

File: trade_file.md [hunk 1]
```diff
@@ -71,8 +71,8 @@ This trade represents a trade between a trading entity and an exchange. For exam
 | `account_id` | Yes |
 | `quantity` | Yes |
 | `price` | Yes |
-| `behalf_of_account_id` | No | `null` |
-| `solicited` | Yes |
+| `behalf_of_account_id` | No | `null` |Preferred if the trade is done on behalf of a Clear Street account ID
```
Comment (tzachshabtay) [inline trade_file.md]: Missing white space `| Preferred`

File: trade_file.md [hunk 2]
```diff
@@ -71,8 +71,8 @@ This trade represents a trade between a trading entity and an exchange. For exam
...
+| `solicited` | No*| `false` | Set the `true` if trade is solicited, otherwise not required
```
Comment (tzachshabtay) [inline trade_file.md]: Missing white space `No*|`

File: trade_file.md [hunk 3]
```diff
@@ -110,8 +110,8 @@ This trade represents a trade between two trading entities. For example, trading
 | `account_id` | Yes |
 | `quantity` | Yes |
 | `price` | Yes |
-| `behalf_of_account_id` | No | `null` |
-| `solicited` | Yes |
+| `behalf_of_account_id` | No | `null` | Preferred if the trade is done on behalf of a Clear Street account 
+| `solicited` | No*| `false` | Set the `true` if trade is solicited, otherwise not required
```
Comment (tzachshabtay) [inline trade_file.md]: Missing white space `No*|`

File: trade_file.md [hunk 4]
```diff
@@ -153,7 +153,7 @@ This trade type is used to facilitate average-price workflows, i.e. averaging ma
 | `quantity` | Yes |
 | `price` | Yes |
 | `behalf_of_account_id` | No | `null` |
-| `solicited` | Yes |
+| `solicited` | No*| `false` | Set the `true` if trade is solicited, otherwise not required
```
Comment (tzachshabtay) [inline trade_file.md]: Missing white space `No*|`

**Review by tzachshabtay — CHANGES_REQUESTED** [review]

**Review by tzachshabtay — APPROVED** [review]

**Review by rtjoseph11 — APPROVED** [review]

**Pull request merged**